client/my-sites/plans/current-plan/index.jsx

```diff
@@ -172,6 +192,18 @@ class CurrentPlan extends Component {
 					/>
 				) }
 
+				{ showExpiryNotice && (
+					<Notice
+						status="is-info"
+						text={ <PlanRenewalNotice purchase={ jetpackLegacyPlanPurchase } withoutLink /> }
+						showDismiss={ false }
+					>
+						<NoticeAction href={ OFFER_RESET_SUPPORT_PAGE }>
+							{ translate( 'More info' ) }
+						</NoticeAction>
+					</Notice>
```

Comment:
I wonder if it would be more optimal to have `Notice` in `PlanRenewalNotice` instead of as it currently is? That would reduce the size of this PR significantly, and also be a bit more accurate in naming `PlanRenewalNotice`.

Reply:
Thing is `PlanRenewalNotice` is also used as a regular paragraph in the description of a product card. But I agree, the name is misleading. I forgot about the `Notice` component when I first named it. I'll do the update!